packages/wdio-appium-service/README.md

```diff
@@ -102,10 +102,10 @@ export const config = {
                 // ...
                 debugLogSpacing: true,
                 platformName: 'iOS',
+                '-p': 1234,
+                '--relaxed-security': true
```

Comment:
I don't think we should document such usage and encourage users to write out the full property name.

Reply:
Understood, but would this apply equally to aliases as well?

Reply:
I am not sure if we should support these, I think if someone e.g. sets `{ p: 1234 }` we can create `['-p', 1234]` out of it. What do you think?

Reply:
I've observed that this approach isn't applicable to all aliases. 

For example: if I have `{ p: 1234 }`, following the formatting, it would result in `['--p', 1234]`, which Appium would accept.

However, consider a scenario where I provide an address in the alias, such as `{ a: 'foo' }`. The formatting would lead to `['--a', 'foo']`, triggering an ambiguous error in Appium.

To address this situation, it becomes necessary to either permit the use of full property names in the args or to consider discontinuing the support by updating the documentation. Please share your opinion.

Reply:
From the user experience point of view I feel like this makes the configuration more clear anyway. What do you think?

Reply:
Certainly, that sounds like a solid plan. I'll integrate those adjustments and make the necessary updates to the documentation.

Reply:
Hi @christian-bromann, I have implemented all the required modifications. Please review them at your convenience.